pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -578,11 +579,14 @@ class Build {
 
             // Get Full Version Output
             String versionPath = "workspace/target/metadata/version.txt"
+            if (buildConfig.ARCHITECTURE.contains("riscv")) {
```

Comment:
I wonder if we can modify this conditional to be more of a generic "If we are cross-compiling" (true for any RIS-V specific references in method names etc. here) but that's 'somethign we can change later as we don't have an obvious way to detect it at the moment so would be quite a bit more work :-)

Reply:
I think that (and @gdams [suggestion above](https://github.com/AdoptOpenJDK/openjdk-build/pull/2238#issuecomment-733633743)) should probably be done under a separate PR once we have this job running for linux and we're happy with it